src/user/tipsandtricks.rst

```diff
@@ -130,3 +130,25 @@ To verify that the correct platform is being used, run the following commands af
 
     python -c "import platform;print(platform.machine())"  # Should print "x86_64"
     echo "CONDA_SUBDIR: $CONDA_SUBDIR"  # Should print "CONDA_SUBDIR: osx-64"
+
+
+.. _installing_packages_for_GPUs_and_CPUs:
+
+Installing CUDA-enabled packages like TensorFlow and PyTorch 
+============================================================
+
+In conda-forge, some packages are available with GPU support. These packages not only take significantly longer to compile and build, but they also result in rather large binaries that users then download. As an effort to maximize accessibility for users with lower connection and/or storage bandwidth, there is an ongoing effort to limit installing packages compiled for GPUs unnecessarily on CPU-only machines by default. This is accomplished by adding a run dependency, ``__cuda``, that detects if the local machine has a GPU. However, this introduces challenges to users who may prefer to still download and use GPU-enabled packages even on a non-GPU machine. For example, login nodes on HPCs often do not have GPUs and their compute counterparts with GPUs often do not have internet access. In this case, a user can override the default setting via the environment variable ``CONDA_CUDA_OVERRIDE`` to install GPU packages on the login node to be used later on the compute node. At the time of writing (February 2022), we have concluded this safe default behavior is best for most of conda-forge users, with an easy override option available and documented. Please let us know if you have thoughts on or issues with this.
+
+In order to override the default behavior, a user can set the environment variable ``CONDA_CUDA_OVERRIDE`` like below to install TensorFlow with GPU support even on a machine with CPU only.
+
+.. code-block:: shell
+
+     CONDA_CUDA_OVERRIDE="11.2" conda install tensorflow==2.7.0=cuda112* -c conda-forge
```

Comment:
it's safer to use `"tensorflow==2.7.0=cuda112*"` otherwise you might interfere with bash / zsh shell expansion.

Reply:
I always forget this about zsh and end up spending a few minutes debugging!